src/screencast_keys/ops.py

```diff
@@ -1507,3 +1539,23 @@ def invoke(self, context, event):
         self.draw_handler_add(context)
         context.window_manager.modal_handler_add(self)
         return {'RUNNING_MODAL'}
+
+
+@BlClassRegistry()
+class SK_OT_WaitForInitialize(bpy.types.Operator):
+    bl_idname = "wm.sk_wait_for_initialize"
+    bl_label = "Screencast Keys Wait For Initialize"
+
+    initialization_handler = None
+
+    def execute(self, context):
+        cls = self.__class__
+        cls.initialization_handler = bpy.types.SpaceView3D.draw_handler_add(cls.intialization_callback, (None, None), 'WINDOW', 'POST_PIXEL')
```

Comment:
[Question]
Does calling `bpy.ops.wm.sk_screencast_keys` work in this place?

Reply:
No. This is the main reason why my changes were created. If there will be direct call of `bpy.ops.wm.sk_screencast_keys`, it won't have any changes beetwen the same call from `load_post_handler`, just a bit deeper. The information about `bpy.context.area` is not loaded at that moment, therefore it must be delayed via some co-routine method.

Reply:
Thanks for the clarification.

Is it possible to add the comment describing here like:

```suggestion
        # If we call bpy.ops.wm.sk_screencast_keys directly, no changes are made because bpy.context.area is not loaded at that moment. 
        # Therefore, we need to call bpy.ops.wm.sk_screencast_keys with delay via co-routine method.
        cls.initialization_handler = bpy.types.SpaceView3D.draw_handler_add(cls.intialization_callback, (None, None), 'WINDOW', 'POST_PIXEL')
```

Reply:
Yeah, I forgot about that. Changed.